boom! testing https://github.com/saschagrunert/git-journal#ff3cf33b426ef9ce1a3c558e1dda7a571b767cd1 against nightly for nightly
boom! running: cargo +nightly build --frozen
boom! creating container for: cargo +nightly build --frozen
boom! running `docker create -v /home/ec2-user/cargobomb/./work/local/test-source/nightly/nightly:/source:ro -v /home/ec2-user/cargobomb/./work/local/cargo-home:/cargo-home:ro -v /home/ec2-user/cargobomb/./work/local/rustup-home:/rustup-home:ro -v /home/ec2-user/cargobomb/./work/local/target-dirs/nightly/nightly:/target:rw -e USER_ID=500 -e CMD=cargo +nightly build --frozen cargobomb`
blam! 1f5ae2c830003035fab7355fa20ea43b5357f13e8b39853c50dd5ba46d82bf06
boom! running `docker start -a 1f5ae2c830003035fab7355fa20ea43b5357f13e8b39853c50dd5ba46d82bf06`
kablam!    Compiling cmake v0.1.21
kablam!    Compiling curl-sys v0.3.10
kablam!    Compiling toml v0.3.1
kablam!    Compiling idna v0.1.0
kablam!    Compiling libz-sys v1.0.13
kablam!    Compiling libssh2-sys v0.2.5
kablam!    Compiling url v1.4.0
kablam!    Compiling libgit2-sys v0.6.7
kablam!    Compiling mowl v1.0.2
kablam!    Compiling chrono v0.3.0
kablam!    Compiling nom v2.1.0
kablam!    Compiling memchr v0.1.11
kablam!    Compiling aho-corasick v0.5.3
kablam!    Compiling thread-id v2.0.0
kablam!    Compiling clap v2.21.1
kablam!    Compiling thread_local v0.2.7
kablam!    Compiling serde_codegen_internals v0.14.1
kablam!    Compiling regex v0.1.80
kablam!    Compiling serde_derive v0.9.11
kablam!    Compiling git2 v0.6.4
kablam!    Compiling git-journal v1.4.2 (file:///source)
kablam!     Finished dev [unoptimized + debuginfo] target(s) in 82.66 secs
boom! running `docker rm -f 1f5ae2c830003035fab7355fa20ea43b5357f13e8b39853c50dd5ba46d82bf06`
blam! 1f5ae2c830003035fab7355fa20ea43b5357f13e8b39853c50dd5ba46d82bf06
boom! running: cargo +nightly test --frozen --no-run
boom! creating container for: cargo +nightly test --frozen --no-run
boom! running `docker create -v /home/ec2-user/cargobomb/./work/local/test-source/nightly/nightly:/source:ro -v /home/ec2-user/cargobomb/./work/local/cargo-home:/cargo-home:ro -v /home/ec2-user/cargobomb/./work/local/rustup-home:/rustup-home:ro -v /home/ec2-user/cargobomb/./work/local/target-dirs/nightly/nightly:/target:rw -e USER_ID=500 -e CMD=cargo +nightly test --frozen --no-run cargobomb`
blam! cc1e0bc7b762ca25c2359cb4628a921ebd88acee268fb412b9ec9a475b10ce31
boom! running `docker start -a cc1e0bc7b762ca25c2359cb4628a921ebd88acee268fb412b9ec9a475b10ce31`
kablam!    Compiling git-journal v1.4.2 (file:///source)
kablam!     Finished dev [unoptimized + debuginfo] target(s) in 13.5 secs
boom! running `docker rm -f cc1e0bc7b762ca25c2359cb4628a921ebd88acee268fb412b9ec9a475b10ce31`
blam! cc1e0bc7b762ca25c2359cb4628a921ebd88acee268fb412b9ec9a475b10ce31
boom! running: cargo +nightly test --frozen
boom! creating container for: cargo +nightly test --frozen
boom! running `docker create -v /home/ec2-user/cargobomb/./work/local/test-source/nightly/nightly:/source:ro -v /home/ec2-user/cargobomb/./work/local/cargo-home:/cargo-home:ro -v /home/ec2-user/cargobomb/./work/local/rustup-home:/rustup-home:ro -v /home/ec2-user/cargobomb/./work/local/target-dirs/nightly/nightly:/target:rw -e USER_ID=500 -e CMD=cargo +nightly test --frozen cargobomb`
blam! 2553d8d30f525b1c6d05c338f0f7d7118a51afe3b665cf0dcc7a757ec3ffa428
boom! running `docker start -a 2553d8d30f525b1c6d05c338f0f7d7118a51afe3b665cf0dcc7a757ec3ffa428`
kablam!     Finished dev [unoptimized + debuginfo] target(s) in 0.0 secs
kablam!      Running /target/debug/deps/git_journal-694e6d055d650f70
blam! 
blam! running 0 tests
blam! 
blam! test result: ok. 0 passed; 0 failed; 0 ignored; 0 measured
blam! 
kablam!      Running /target/debug/deps/gitjournal-968a056d20e14903
blam! 
blam! running 42 tests
blam! test config::tests::config_load_invalid_1 ... ok
blam! test config::tests::config_load_err ... ok
blam! test config::tests::config_load_invalid_2 ... ok
blam! test config::tests::config_load_invalid_3 ... ok
blam! test config::tests::config_save_err ... ok
blam! test config::tests::config_save_and_load_ok ... FAILED
blam! test parser::tests::parse_commit_failure_1 ... ok
blam! test parser::tests::parse_commit_failure_2 ... ok
blam! test parser::tests::parse_commit_failure_3 ... ok
blam! test parser::tests::parse_commit_ok_1 ... FAILED
blam! test tests::generate_template_1 ... FAILED
blam! test tests::install_git_hook ... FAILED
blam! test tests::new ... FAILED
blam! test tests::parse_and_print_log_1 ... FAILED
blam! test tests::parse_and_print_log_2 ... FAILED
blam! test tests::parse_and_print_log_3 ... FAILED
blam! test tests::parse_and_print_log_4 ... FAILED
blam! test tests::parse_and_print_log_5 ... FAILED
blam! test tests::parse_and_print_log_6 ... FAILED
blam! test parser::tests::parse_commit_ok_3 ... FAILED
blam! test tests::prepare_message_failure_1 ... FAILED
blam! test tests::path_failure ... ok
blam! test tests::prepare_message_failure_2 ... FAILED
blam! test tests::prepare_message_success_1 ... FAILED
blam! test tests::prepare_message_success_2 ... FAILED
blam! test tests::prepare_message_success_3 ... FAILED
blam! test tests::prepare_message_success_4 ... FAILED
blam! test tests::setup_failed ... FAILED
blam! test tests::setup_succeed ... FAILED
blam! test tests::verify_commit_msg_paragraph_failure_1 ... FAILED
blam! test tests::verify_commit_msg_paragraph_failure_2 ... FAILED
blam! test tests::verify_commit_msg_paragraph_failure_3 ... FAILED
blam! test tests::verify_commit_msg_summary_failure_2 ... FAILED
blam! test tests::verify_commit_msg_summary_failure_3 ... FAILED
blam! test tests::verify_commit_msg_summary_failure_tag ... FAILED
blam! test tests::verify_commit_msg_summary_success_1 ... FAILED
blam! test tests::verify_commit_msg_summary_success_2 ... FAILED
blam! test tests::verify_commit_msg_summary_success_3 ... FAILED
blam! test tests::verify_commit_msg_summary_success_4 ... FAILED
blam! test tests::verify_commit_msg_summary_failure_1 ... FAILED
blam! test parser::tests::parse_commit_ok_2 ... FAILED
blam! test parser::tests::parse_commit_ok_4 ... FAILED
blam! 
blam! failures:
blam! 
blam! ---- config::tests::config_save_and_load_ok stdout ----
blam! 	thread 'config::tests::config_save_and_load_ok' panicked at 'assertion failed: config.save_default_config(".").is_ok()', src/config.rs:163
blam! note: Run with `RUST_BACKTRACE=1` for a backtrace.
blam! 
blam! ---- parser::tests::parse_commit_ok_1 stdout ----
blam! 	thread 'parser::tests::parse_commit_ok_1' panicked at 'called `Option::unwrap()` on a `None` value', /checkout/src/libcore/option.rs:329
blam! 
blam! ---- tests::generate_template_1 stdout ----
blam! 	thread 'tests::generate_template_1' panicked at 'called `Result::unwrap()` on an `Err` value: Error(Io(Error { repr: Os { code: 2, message: "No such file or directory" } }), State { next_error: None, backtrace: None })', /checkout/src/libcore/result.rs:859
blam! 
blam! ---- tests::install_git_hook stdout ----
blam! 	Logging failed: Could not create terminal.
blam! thread 'tests::install_git_hook' panicked at 'called `Result::unwrap()` on an `Err` value: Error(Git(Error { code: -3, klass: 6, message: "could not find repository from \'/\'" }), State { next_error: None, backtrace: None })', /checkout/src/libcore/result.rs:859
blam! 
blam! ---- tests::new stdout ----
blam! 	Logging failed: Could not create terminal.
blam! thread 'tests::new' panicked at 'assertion failed: GitJournal::new(".").is_ok()', src/lib.rs:649
blam! 
blam! ---- tests::parse_and_print_log_1 stdout ----
blam! 	Logging failed: Could not create terminal.
blam! thread 'tests::parse_and_print_log_1' panicked at 'called `Result::unwrap()` on an `Err` value: Error(Io(Error { repr: Os { code: 2, message: "No such file or directory" } }), State { next_error: None, backtrace: None })', /checkout/src/libcore/result.rs:859
blam! 
blam! ---- tests::parse_and_print_log_2 stdout ----
blam! 	Logging failed: Could not create terminal.
blam! thread 'tests::parse_and_print_log_2' panicked at 'called `Result::unwrap()` on an `Err` value: Error(Io(Error { repr: Os { code: 2, message: "No such file or directory" } }), State { next_error: None, backtrace: None })', /checkout/src/libcore/result.rs:859
blam! 
blam! ---- tests::parse_and_print_log_3 stdout ----
blam! 	Logging failed: Could not create terminal.
blam! thread 'tests::parse_and_print_log_3' panicked at 'called `Result::unwrap()` on an `Err` value: Error(Io(Error { repr: Os { code: 2, message: "No such file or directory" } }), State { next_error: None, backtrace: None })', /checkout/src/libcore/result.rs:859
blam! 
blam! ---- tests::parse_and_print_log_4 stdout ----
blam! 	Logging failed: Could not create terminal.
blam! thread 'tests::parse_and_print_log_4' panicked at 'called `Result::unwrap()` on an `Err` value: Error(Io(Error { repr: Os { code: 2, message: "No such file or directory" } }), State { next_error: None, backtrace: None })', /checkout/src/libcore/result.rs:859
blam! 
blam! ---- tests::parse_and_print_log_5 stdout ----
blam! 	Logging failed: Could not create terminal.
blam! thread 'tests::parse_and_print_log_5' panicked at 'called `Result::unwrap()` on an `Err` value: Error(Io(Error { repr: Os { code: 2, message: "No such file or directory" } }), State { next_error: None, backtrace: None })', /checkout/src/libcore/result.rs:859
blam! 
blam! ---- tests::parse_and_print_log_6 stdout ----
blam! 	Logging failed: Could not create terminal.
blam! thread 'tests::parse_and_print_log_6' panicked at 'called `Result::unwrap()` on an `Err` value: Error(Io(Error { repr: Os { code: 2, message: "No such file or directory" } }), State { next_error: None, backtrace: None })', /checkout/src/libcore/result.rs:859
blam! 
blam! ---- parser::tests::parse_commit_ok_3 stdout ----
blam! 	thread 'parser::tests::parse_commit_ok_3' panicked at 'called `Option::unwrap()` on a `None` value', /checkout/src/libcore/option.rs:329
blam! 
blam! ---- tests::prepare_message_failure_1 stdout ----
blam! 	Logging failed: Could not create terminal.
blam! thread 'tests::prepare_message_failure_1' panicked at 'called `Result::unwrap()` on an `Err` value: Error(Git(Error { code: -3, klass: 6, message: "could not find repository from \'/\'" }), State { next_error: None, backtrace: None })', /checkout/src/libcore/result.rs:859
blam! 
blam! ---- tests::prepare_message_failure_2 stdout ----
blam! 	Logging failed: Could not create terminal.
blam! thread 'tests::prepare_message_failure_2' panicked at 'called `Result::unwrap()` on an `Err` value: Error(Git(Error { code: -3, klass: 6, message: "could not find repository from \'/\'" }), State { next_error: None, backtrace: None })', /checkout/src/libcore/result.rs:859
blam! 
blam! ---- tests::prepare_message_success_1 stdout ----
blam! 	Logging failed: Could not create terminal.
blam! thread 'tests::prepare_message_success_1' panicked at 'called `Result::unwrap()` on an `Err` value: Error(Git(Error { code: -3, klass: 6, message: "could not find repository from \'/\'" }), State { next_error: None, backtrace: None })', /checkout/src/libcore/result.rs:859
blam! 
blam! ---- tests::prepare_message_success_2 stdout ----
blam! 	Logging failed: Could not create terminal.
blam! thread 'tests::prepare_message_success_2' panicked at 'called `Result::unwrap()` on an `Err` value: Error(Git(Error { code: -3, klass: 6, message: "could not find repository from \'/\'" }), State { next_error: None, backtrace: None })', /checkout/src/libcore/result.rs:859
blam! 
blam! ---- tests::prepare_message_success_3 stdout ----
blam! 	Logging failed: Could not create terminal.
blam! thread 'tests::prepare_message_success_3' panicked at 'called `Result::unwrap()` on an `Err` value: Error(Git(Error { code: -3, klass: 6, message: "could not find repository from \'/\'" }), State { next_error: None, backtrace: None })', /checkout/src/libcore/result.rs:859
blam! 
blam! ---- tests::prepare_message_success_4 stdout ----
blam! 	Logging failed: Could not create terminal.
blam! thread 'tests::prepare_message_success_4' panicked at 'called `Result::unwrap()` on an `Err` value: Error(Git(Error { code: -3, klass: 6, message: "could not find repository from \'/\'" }), State { next_error: None, backtrace: None })', /checkout/src/libcore/result.rs:859
blam! 
blam! ---- tests::setup_failed stdout ----
blam! 	Logging failed: Could not create terminal.
blam! thread 'tests::setup_failed' panicked at 'assertion failed: journal.is_ok()', src/lib.rs:669
blam! 
blam! ---- tests::setup_succeed stdout ----
blam! 	Logging failed: Could not create terminal.
blam! thread 'tests::setup_succeed' panicked at 'assertion failed: journal.is_ok()', src/lib.rs:661
blam! 
blam! ---- tests::verify_commit_msg_paragraph_failure_1 stdout ----
blam! 	Logging failed: Could not create terminal.
blam! thread 'tests::verify_commit_msg_paragraph_failure_1' panicked at 'called `Result::unwrap()` on an `Err` value: Error(Git(Error { code: -3, klass: 6, message: "could not find repository from \'/\'" }), State { next_error: None, backtrace: None })', /checkout/src/libcore/result.rs:859
blam! 
blam! ---- tests::verify_commit_msg_paragraph_failure_2 stdout ----
blam! 	Logging failed: Could not create terminal.
blam! thread 'tests::verify_commit_msg_paragraph_failure_2' panicked at 'called `Result::unwrap()` on an `Err` value: Error(Git(Error { code: -3, klass: 6, message: "could not find repository from \'/\'" }), State { next_error: None, backtrace: None })', /checkout/src/libcore/result.rs:859
blam! 
blam! ---- tests::verify_commit_msg_paragraph_failure_3 stdout ----
blam! 	Logging failed: Could not create terminal.
blam! thread 'tests::verify_commit_msg_paragraph_failure_3' panicked at 'called `Result::unwrap()` on an `Err` value: Error(Git(Error { code: -3, klass: 6, message: "could not find repository from \'/\'" }), State { next_error: None, backtrace: None })', /checkout/src/libcore/result.rs:859
blam! 
blam! ---- tests::verify_commit_msg_summary_failure_2 stdout ----
blam! 	Logging failed: Could not create terminal.
blam! thread 'tests::verify_commit_msg_summary_failure_2' panicked at 'called `Result::unwrap()` on an `Err` value: Error(Git(Error { code: -3, klass: 6, message: "could not find repository from \'/\'" }), State { next_error: None, backtrace: None })', /checkout/src/libcore/result.rs:859
blam! 
blam! ---- tests::verify_commit_msg_summary_failure_3 stdout ----
blam! 	Logging failed: Could not create terminal.
blam! thread 'tests::verify_commit_msg_summary_failure_3' panicked at 'called `Result::unwrap()` on an `Err` value: Error(Git(Error { code: -3, klass: 6, message: "could not find repository from \'/\'" }), State { next_error: None, backtrace: None })', /checkout/src/libcore/result.rs:859
blam! 
blam! ---- tests::verify_commit_msg_summary_failure_tag stdout ----
blam! 	Logging failed: Could not create terminal.
blam! thread 'tests::verify_commit_msg_summary_failure_tag' panicked at 'called `Result::unwrap()` on an `Err` value: Error(Io(Error { repr: Os { code: 2, message: "No such file or directory" } }), State { next_error: None, backtrace: None })', /checkout/src/libcore/result.rs:859
blam! 
blam! ---- tests::verify_commit_msg_summary_success_1 stdout ----
blam! 	Logging failed: Could not create terminal.
blam! thread 'tests::verify_commit_msg_summary_success_1' panicked at 'called `Result::unwrap()` on an `Err` value: Error(Git(Error { code: -3, klass: 6, message: "could not find repository from \'/\'" }), State { next_error: None, backtrace: None })', /checkout/src/libcore/result.rs:859
blam! 
blam! ---- tests::verify_commit_msg_summary_success_2 stdout ----
blam! 	Logging failed: Could not create terminal.
blam! thread 'tests::verify_commit_msg_summary_success_2' panicked at 'called `Result::unwrap()` on an `Err` value: Error(Git(Error { code: -3, klass: 6, message: "could not find repository from \'/\'" }), State { next_error: None, backtrace: None })', /checkout/src/libcore/result.rs:859
blam! 
blam! ---- tests::verify_commit_msg_summary_success_3 stdout ----
blam! 	Logging failed: Could not create terminal.
blam! thread 'tests::verify_commit_msg_summary_success_3' panicked at 'called `Result::unwrap()` on an `Err` value: Error(Git(Error { code: -3, klass: 6, message: "could not find repository from \'/\'" }), State { next_error: None, backtrace: None })', /checkout/src/libcore/result.rs:859
blam! 
blam! ---- tests::verify_commit_msg_summary_success_4 stdout ----
blam! 	Logging failed: Could not create terminal.
blam! thread 'tests::verify_commit_msg_summary_success_4' panicked at 'called `Result::unwrap()` on an `Err` value: Error(Git(Error { code: -3, klass: 6, message: "could not find repository from \'/\'" }), State { next_error: None, backtrace: None })', /checkout/src/libcore/result.rs:859
blam! 
blam! ---- tests::verify_commit_msg_summary_failure_1 stdout ----
blam! 	Logging failed: Could not create terminal.
blam! thread 'tests::verify_commit_msg_summary_failure_1' panicked at 'called `Result::unwrap()` on an `Err` value: Error(Git(Error { code: -3, klass: 6, message: "could not find repository from \'/\'" }), State { next_error: None, backtrace: None })', /checkout/src/libcore/result.rs:859
blam! 
blam! ---- parser::tests::parse_commit_ok_2 stdout ----
blam! 	thread 'parser::tests::parse_commit_ok_2' panicked at 'called `Option::unwrap()` on a `None` value', /checkout/src/libcore/option.rs:329
blam! 
blam! ---- parser::tests::parse_commit_ok_4 stdout ----
blam! 	thread 'parser::tests::parse_commit_ok_4' panicked at 'called `Option::unwrap()` on a `None` value', /checkout/src/libcore/option.rs:329
blam! 
blam! 
blam! failures:
blam!     config::tests::config_save_and_load_ok
blam!     parser::tests::parse_commit_ok_1
blam!     parser::tests::parse_commit_ok_2
blam!     parser::tests::parse_commit_ok_3
blam!     parser::tests::parse_commit_ok_4
blam!     tests::generate_template_1
blam!     tests::install_git_hook
blam!     tests::new
blam!     tests::parse_and_print_log_1
blam!     tests::parse_and_print_log_2
blam!     tests::parse_and_print_log_3
blam!     tests::parse_and_print_log_4
blam!     tests::parse_and_print_log_5
blam!     tests::parse_and_print_log_6
kablam! error: test failed
blam!     tests::prepare_message_failure_1
blam!     tests::prepare_message_failure_2
blam!     tests::prepare_message_success_1
blam!     tests::prepare_message_success_2
blam!     tests::prepare_message_success_3
blam!     tests::prepare_message_success_4
blam!     tests::setup_failed
blam!     tests::setup_succeed
blam!     tests::verify_commit_msg_paragraph_failure_1
blam!     tests::verify_commit_msg_paragraph_failure_2
blam!     tests::verify_commit_msg_paragraph_failure_3
blam!     tests::verify_commit_msg_summary_failure_1
blam!     tests::verify_commit_msg_summary_failure_2
blam!     tests::verify_commit_msg_summary_failure_3
blam!     tests::verify_commit_msg_summary_failure_tag
blam!     tests::verify_commit_msg_summary_success_1
blam!     tests::verify_commit_msg_summary_success_2
blam!     tests::verify_commit_msg_summary_success_3
blam!     tests::verify_commit_msg_summary_success_4
blam! 
blam! test result: FAILED. 9 passed; 33 failed; 0 ignored; 0 measured
blam! 
boom! running `docker rm -f 2553d8d30f525b1c6d05c338f0f7d7118a51afe3b665cf0dcc7a757ec3ffa428`
blam! 2553d8d30f525b1c6d05c338f0f7d7118a51afe3b665cf0dcc7a757ec3ffa428
